api/sip/__init__.py

```diff
@@ -86,7 +101,10 @@ def info_to_patrondata(cls, info):
         if 'personal_name' in info:
             patrondata.personal_name = info['personal_name']
         if 'fee_amount' in info:
-            patrondata.fines = info['fee_amount']
+            fines = info['fee_amount']
+        else:
+            fines = '0'
+        patrondata.fines = MoneyUtility.parse(fines)
```

Comment:
This is the code that actually makes sure that the incoming string from the SIP2 message is converted into an amount of money. The code above turns the amount of money into a number.